src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -47,6 +47,19 @@ public IConverterManager ConverterManager
         }
 
         /// <summary>
+        /// Creating a type filter predicate around another rule. Filter doubles as a validator if it throws.  
+        /// </summary>
+        /// <param name="predicate">type predicate. Only apply inner rule if this predicate as applied to the user parameter type is true. </param>
+        /// <param name="innerRule">Inner rule. </param>
+        /// <returns></returns>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic")]
```

Comment:
I have a strong preference for inlining them -  a) their ugly presence is a motivator to fix it; b) the global file easily gets out of sync with stale entries. 


---
In reply to: [63629010](https://github.com/Azure/azure-webjobs-sdk/pull/726#discussion_r63629010) [](ancestors = 63629010)